src/openapi.json

```diff
@@ -1386,7 +1406,7 @@
 							}
 						}
 					},
-					"409": {
+					"422": {
```

Comment:
What do you think of making this description a bit broader -- a 422 can be provided for other validation errors as well (e.g. invalid data type for a field, invalid value for a field, or in this case, an invalid reference).

Even if the only possible reason for this specific endpoint is the fk conflict, I would make the message more user-focused (rather than db terminology) sugh as "the referenced user entity didn't exist".  That said, I think it might be more long-term-sustainable to keep it high level here / rely on the body of the error to convey exactly what invalid data was provided.

Reply:
@slifty I think that's reasonable, I like the move towards more user-focused language. Check out latest push here, 
```"422": {
						"description": "The proposal references entities that do not exist in the database.",
						"content": {
							"application/json": {
								"schema": {
									"$ref": "#/components/schemas/PdcError"
								}
							}
						}
					}
```
```
"422": {
						"description": "The proposal version references entities that do not exist in the database.",
						"content": {
							"application/json": {
								"schema": {
									"$ref": "#/components/schemas/PdcError"
								}
							}
						}
					}
```
```
"422": {
						"description": "The application references entities that do not exist in the database.",
						"content": {
							"application/json": {
								"schema": {
									"$ref": "#/components/schemas/PdcError"
								}
							}
						}
					}
					```

Reply:
Looks great!

So... spoiler alert: I think we are going to want to update *all* validation errors to return 422 (400 is actually for signaling that the structure of the data is incorrect / unrecognized, not "we recognized it, but this string didn't match the pattern defined by business rules")

That said, for now we don't do that so this documentation is correct!